### DISCUSSION TITLE
OPC-UA-Client "SUBSCRIBE" + MYSQL + MINDCONNECT is not working

### DISCUSSION THREAD

**Discussion by Olex94** [mindsphere/node-red-contrib-mindconnect#177] [Q&A]
Hi,

I want to store my OPC-UA Values in a Database, and Send them via Mindconnect to Mindsphere. 
With the OPC-UA Client "Read" its working, but then the Database gets every x Second a value, so we want to use the "Subscribe" function.
The Request via Mindconnect ist succesful, but there is no Data in Mindsphere.
Can Somebody help me?

![image](https://github.com/mindsphere/node-red-contrib-mindconnect/assets/131179141/e6ea15ce-5309-463e-ae99-b27c556d22d7)

[Flow without SHA.txt](https://github.com/mindsphere/node-red-contrib-mindconnect/files/11463841/Flow.without.SHA.txt)

**Comment by sn0wcat** [comment]
please use agent diagnostic tool on your insights hub dashboard to check if the data is arriving...see also 
https://opensource.mindsphere.io/docs/mindconnect-nodejs/troubleshooting.html
